src/conn_handler.c

```diff
@@ -136,9 +132,14 @@ static int stream_formatter(FILE *pipe, void *data, metric_type type, char *name
             STREAM("%s%s.upper|%f|%lld\n", prefix, name, timer_max(&t->tm));
             STREAM("%s%s.count|%lld|%lld\n", prefix, name, timer_count(&t->tm));
             STREAM("%s%s.stdev|%f|%lld\n", prefix, name, timer_stddev(&t->tm));
-            STREAM("%s%s.median|%f|%lld\n", prefix, name, timer_query(&t->tm, 0.5));
-            STREAM("%s%s.p95|%f|%lld\n", prefix, name, timer_query(&t->tm, 0.95));
-            STREAM("%s%s.p99|%f|%lld\n", prefix, name, timer_query(&t->tm, 0.99));
+            for (i=0; i < GLOBAL_CONFIG->num_quantiles; i++) {
+                if (GLOBAL_CONFIG->quantiles[i] == 0.5) {
+                    STREAM("%s%s.median|%f|%lld\n", prefix, name, timer_query(&t->tm, 0.5));
```

Comment:
Adding p50 gives compatilbity with statsd, which is nice if you're moving from that to statsite.